src/content/it/2020/resource-hints.md

```diff
@@ -0,0 +1,461 @@
+---
+part_number: IV
+chapter_number: 21
+title: Resource Hints
+description: Resource Hints chapter of the 2020 Web Almanac covering usage of dns-prefetch, preconnect, preload, prefetch, Priority Hints, and native lazy loading.
+authors: [Zizzamia]
+reviewers: [jessnicolet, pmeenan, giopunt, mgechev, notwillk]
+analysts: [khempenius]
+translators: []
```

Comment:
You get translators credit when you translate a chapter!

```suggestion
translators: [chefleo]
```

Also can youn add your details to `config/2020.json`? Alphabetical by first name.